packages/nest/README.md

```diff
@@ -103,7 +116,7 @@ export class OpenFeatureController {
 }
 ```
 
-#### Injecting the default and a named client into a service:
+It is also possible to inject the default and a named OpenFeature client into a service via it's constructor with Nest dependency injection system.
```

Comment:
```suggestion
It is also possible to inject the default and named OpenFeature clients into a service via Nest dependency injection system.
```

Reply:
I think "the default and named OpenFeature clients" should read "the default or named OpenFeature clients".